### PR TITLE
Cortana with Outlook desktop is supported

### PR DESCRIPTION
**Pull request by joinimran**
Cortana with outlook is now supported and I have updated the document accordingly.

Problem: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/381

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/misc/cortana-integration.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ If you don't want Cortana to access your organization's data, you can turn Corta
 
 There are a few things to be aware of before you start using Cortana in Office 365. If you run into issues, check out [these troubleshooting tips](https://go.microsoft.com/fwlink/?LinkId=620763).
   
-- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (Support for using Cortana with Outlook on the desktop is planned for a later release.)
+- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (You can also use Cortana with Outlook on the desktop.)
```
Comment (JohanFreelancer9) [inline Office365-Admin/misc/cortana-integration.md]: Copy checked - no changes.

Comment (nenonix) [inline Office365-Admin/misc/cortana-integration.md]: I Suggest:
You can also use Cortana with **the Outlook desktop app.**
I suggest losing the (  ) too.

Comment (mypil) [inline Office365-Admin/misc/cortana-integration.md]:
```suggestion
- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.
```

Comment (mypil) [inline Office365-Admin/misc/cortana-integration.md]:
```suggestion
- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.
```

**Comment by mypil** [comment]
> "You can add your Outlook or Office 365 account as a connected service to Cortana and add reminders and tasks to your lists in Microsoft To-Do. You can also use Cortana to read and search messages in the Outlook desktop app.

@joinimran - I will create a copy/edit for this that you can merge into this PR.

File: Office365-Admin/misc/cortana-integration.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you don't want Cortana to access your organization's data, you can turn Corta
 
 There are a few things to be aware of before you start using Cortana in Office 365. If you run into issues, check out [these troubleshooting tips](https://go.microsoft.com/fwlink/?LinkId=620763).
   
-- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (Support for using Cortana with Outlook on the desktop is planned for a later release.)
+- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.
```
Comment (mypil) [inline Office365-Admin/misc/cortana-integration.md]:
```suggestion
- You can add your Outlook or Office 365 account as a connected service to Cortana and add reminders and tasks to your lists in Microsoft To-Do. You can also use Cortana to read and search messages in the Outlook desktop app.
```

**Comment by dianef77** [comment]
@kwekuako please review and decide whether to merge

**Pull request merged**